src/components/DoctorCard/Shared/DoctorLinks.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+
+import { Typography } from '@mui/material';
+import * as Icons from 'components/Shared/Icons';
+
+import * as Styled from '../styles';
+import * as Links from './Links';
+
+const DoctorLinks = function DoctorLinks({ links, isWebsite }) {
+  const phoneLinks = links.map((phone, index, arr) => {
+    const href = isWebsite ? phone.href : `tel:${phone?.trim()}`;
+    const text = isWebsite ? phone.host : phone;
+    const key = isWebsite ? `website-${phone.href}` : phone;
+    return (
+      <Links.ConditionalLink
+        key={key}
+        to={phone && href}
+        self={isWebsite ? undefined : true}
+        variant="body1"
+      >
+        {text}
+        <Typography component="span" variant="body1">
+          {index !== arr.length - 1 && ', '}
+        </Typography>
```

Comment:
`
    const isLastLink = index === arr.length - 1;
`

```suggestion
        {!isLastLink && (
          <Typography component="span" variant="body1">
            {', '}
          </Typography>
        )}
```

Maby a bit nitpicky... to avoid empty span